apps/base/dev/tasks.py

```diff
@@ -35,6 +38,25 @@ def fake_data():
     fdg.run()
 
 
+@dev_cli.command("enable_cfp")
+def enable_cfp():
+    for flag in ["LINE_UP", "CFP"]:
+        if not FeatureFlag.query.get(flag):
+            db.session.add(FeatureFlag(feature=flag, enabled=True))
+
+    signup_state = SiteState.query.filter_by(name="signup_state").one_or_none()
+    if not signup_state:
+        db.session.add(SiteState("signup_state", "issue-lottery-tickets"))
+    else:
+        signup_state.state = "issue-lottery-tickets"
+
+    db.session.commit()
+    db.session.flush()
```

Comment:
There are a few examples of this pattern. You shouldn't ever need to flush immediately after a commit, was something going wrong?